docs/core/config/auth/schemes.md

```diff
@@ -119,7 +119,6 @@ exceptions:
 
 | Database | Default |
 | -------- | ------- |
-| [[link,auth_ldap]] | See `default_pass_scheme` setting in `dovecot-ldap.conf.ext`. |
 | [[link,auth_passwd_file]] | `CRYPT` is used by default, but can be changed
```

Comment:
Delete this auth_passwd_file also. It uses the same passdb_default_password_scheme now.